bbb-common-web/sql/create.sql

```diff
@@ -0,0 +1,64 @@
+CREATE DATABASE bbb;

```

Comment:
I am for following the naming conventions of FlywayDB and enabling the automatic discovery. All explained here https://flywaydb.org/documentation/concepts/migrations.html#naming-1